src/gui/shareusergroupwidget.cpp

```diff
@@ -646,37 +646,48 @@ void ShareUserLine::loadAvatar()
     _ui->avatar->setMaximumWidth(avatarSize);
     _ui->avatar->setAlignment(Qt::AlignCenter);
 
+    setDefaultAvatar(avatarSize);
+
+    /* Start the network job to fetch the avatar data.
+     *
+     * Currently only regular users can have avatars.
+     */
+    if (_share->getShareWith()->type() == Sharee::User) {
+        auto *job = new AvatarJob(_share->account(), _share->getShareWith()->shareWith(), avatarSize, this);
```

Comment:
auto with * is redundant

Reply:
I usually set it like this to emphasize the fact that a raw pointer is expected rather than a smart one

Reply:
Coul be. Well, just a copy-paste, really.